notebook.rst

```diff
@@ -1,135 +1,12 @@
 .. _notebook:
    
 ======================  
-The IPython Notebook 
+The IPython/Jupyter Notebook 
```

Comment:
Yeah, that was my thought -- I wanted to still make this be the most likely hit for "ipython notebook". But I could probably be convinced to remove the "IPython".